server/common/database/seeds/users.js

```diff
@@ -21,11 +21,11 @@ const users = [{
 times(10, i => {
   const suffix = i || '';
   users.push({
-    first_name: `Student ${suffix}`,
+    first_name: `Learner ${suffix}`,
```

Comment:
Since you're already touching this, we have [validation](https://github.com/ExtensionEngine/boutique/blob/develop/client/admin/components/users/UserDialog.vue#L38) for the `firstName` that this does not pass which creates issues when editing users.
Can you, please, make the seed comply with it?